fw/t/unit/test_http_parser.c

```diff
@@ -146,10 +147,14 @@ split_and_parse_n(unsigned char *str, int type, size_t len, size_t chunk_size)
 			chunk_size = len - pos;
 		TEST_DBG3("split: len=%zu pos=%zu\n",
 			  len, pos);
+		chunk = kmalloc(chunk_size, GFP_KERNEL);
```

Comment:
You can allocate once per call rather than in loop

Reply:
Detached chunk mode for tests has already been merged.